src/sync.js

```diff
@@ -360,9 +360,6 @@ SyncApi.prototype.getSyncState = function() {
  * Main entry point
  */
 SyncApi.prototype.sync = function() {
-    debuglog("SyncApi.sync: starting with sync token " +
```

Comment:
It felt somewhat redundant/noisy, having added https://github.com/matrix-org/matrix-js-sdk/pull/361/files#diff-b5ce4044fcc6267228c6281fd7483b4cR511.